File: src/src/gestorAplicacion/libreria/Ejemplar.java
package gestorAplicacion.libreria;

import java.io.Serializable;

public class Ejemplar implements Serializable {
    private int id;
    private EstadoEjemplar estadoEjemplar;

    public Ejemplar(int id, EstadoEjemplar estadoEjemplar) {
        this.id = id;
        this.estadoEjemplar = estadoEjemplar;
    }

    public int getId() {
        return id;
    }

    public EstadoEjemplar getEstadoEjemplar() {
        return estadoEjemplar;
    }


}

File: Python/gestorAplicacion/libreria/ejemplar.py
from estadoEjemplar import EstadoEjemplar

class Ejemplar():
    def __init__(self, identificador: int, estadoEjemplar: EstadoEjemplar):
        self._identificador = identificador
        self._estadoEjemplar = estadoEjemplar

    def getIdentificador(self) -> int:
        return self._identificador

    def getEstadoEjemplar(self) -> EstadoEjemplar:
        return self._estadoEjemplar
